x-pack/plugins/security_solution/public/common/hooks/eql/helpers.test.ts

```diff
@@ -91,53 +90,53 @@ describe('eql/helpers', () => {
   describe('calculateBucketForDay', () => {
     test('returns 0 if two dates are equivalent', () => {
       const diff = calculateBucketForDay(
-        Number(dateMath.parse('now')?.format('x')),
-        Number(dateMath.parse('now')?.format('x'))
+        Date.parse('2020-02-20T05:57:54.037Z'),
+        Date.parse('2020-02-20T05:57:54.037Z')
       );
 
       expect(diff).toEqual(0);
     });
 
     test('returns 1 if event occurred within 60 minutes of "now"', () => {
       const diff = calculateBucketForDay(
-        Number(dateMath.parse('now-40m')?.format('x')),
-        Number(dateMath.parse('now')?.format('x'))
+        Date.parse('2020-02-20T05:17:54.037Z'),
+        Date.parse('2020-02-20T05:57:54.037Z')
       );
 
       expect(diff).toEqual(1);
     });
 
     test('returns 2 if event occurred 60-120 minutes from "now"', () => {
       const diff = calculateBucketForDay(
-        Number(dateMath.parse('now-120m')?.format('x')),
-        Number(dateMath.parse('now')?.format('x'))
+        Date.parse('2020-02-20T03:57:54.037Z'),
+        Date.parse('2020-02-20T05:57:54.037Z')
       );
 
       expect(diff).toEqual(2);
     });
 
     test('returns 3 if event occurred 120-180 minutes from "now', () => {
       const diff = calculateBucketForDay(
-        Number(dateMath.parse('now-121m')?.format('x')),
-        Number(dateMath.parse('now')?.format('x'))
+        Date.parse('2020-02-20T03:56:54.037Z'),
+        Date.parse('2020-02-20T05:57:54.037Z')
       );
 
       expect(diff).toEqual(3);
     });
 
     test('returns 4 if event occurred 180-240 minutes from "now', () => {
       const diff = calculateBucketForDay(
-        Number(dateMath.parse('now-220m')?.format('x')),
-        Number(dateMath.parse('now')?.format('x'))
+        Date.parse('2020-02-20T02:15:54.037Z'),
+        Date.parse('2020-02-20T05:57:54.037Z')
```

Comment:
I think this is 222 minutes... does that matter? Also, is the test description correct?

Reply:
I updated the description a bit. That test is just looking for the difference in time to be between 180-240 minutes, so 222 is fine. Let me know if I missed something with the descriptions!

Reply:
This is perfect, thank you!